.github/workflows/build-container.yml

```diff
@@ -0,0 +1,67 @@
+name: Release ingress-netperf container image
+
+on:
+  pull_request:
+    paths:
+      - 'containers/Containerfile' # Trigger only when Dockerfile changes in a pull request
+
+env:
+  CONTAINER_REGISTRY: ${{ github.repository_owner == 'cloud-bulldozer' && 'quay.io' }}
+  RHEL_VERSION: ubi9
+  GO_VER: 1.19
+
+jobs:
+
+  build-and-push-container-images:
+    name: Build and Push Container Images
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+    
+    steps:
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup docker-buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to the ${{ env.CONTAINER_REGISTRY }} Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.CONTAINER_REGISTRY }}
+          username: ${{ env.CONTAINER_REGISTRY == 'quay.io' && secrets.QUAY_USER || github.actor }}
+          password: ${{ env.CONTAINER_REGISTRY == 'quay.io' && secrets.QUAY_TOKEN || secrets.GITHUB_TOKEN }
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.CONTAINER_REGISTRY }}/${{ github.repository_owner }}/netperf
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
+
+      - name: Build and push multi-arch Image
+        id: push
+        uses: docker/build-push-action@v3
+        with:
+          file: containers/Dockerfile
+          platforms: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+          tags: ${{ steps.meta.outputs.tags }}
+          push: ${{ github.event_name != 'pull_request' }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            RHEL_VERSION=${{ env.RHEL_VERSION }}
+            GO_VER=${{ env.GO_VER }}
+            GO_TAGS=
```

Comment:
The intent of not triggering this as a Makefile's target 
is users could trigger targets and might end up building multi-arch images locally, even if not intended for.

Since this is a very specific action triggered on `containers/**` I kept it in GHA instead

Reply:
I prefer to have the logic in the Makefile, the less stuff GHA does the better

Reply:
I realized I will have to update/replace the docker with podman stuff.
So GHA Workflow will have to get updated.

Reply:
Ok, I have updated this now, ptal!